app/javascript/components/requests/Receipt.jsx

```diff
@@ -66,47 +39,89 @@ class Receipt extends React.Component {
       )
   }
 
+  renderStatus() {
+    if (!this.state.request.open) {
+      if (this.state.request.receipt) {
+        const purchaseType = this.state.request.receipt.transaction_type == "purchase" ?
+          "sale" : "rental";
+        return (
+          <p className="green">
+            Confirmed {purchaseType} on {new Date(this.state.request.receipt.purchase_date).toLocaleDateString()}
+          </p>
+        )
+      } else {
+        return (
+          <p className="gray">Closed on {new Date(this.state.request.updated_at).toLocaleDateString()}</p>
+        )
+      }
+    } else {
+      return (
+        <p className="dark-gray">Pending {this.state.request.types}</p>
+      )
+    }
+  }
+
   render() {
-    const request = this.state.request;
-    const id = request.id;
-    const thumbnail_url = request.work.thumbnail ? request.work.thumbnail : "https://cdn0.iconfinder.com/data/icons/typicons-2/24/image-128.png";
-    const closed_timestamps = new Date(request.updated_at).toLocaleDateString();
+    const isRental = this.state.request.receipt.transaction_type == "rental";
+
+    const startDate = isRental ? 
+      new Date(this.state.request.receipt.start_date).toLocaleDateString() : "";
+    const endDate = isRental ? 
+      new Date(this.state.request.receipt.end_date).toLocaleDateString() : "";
 
     return (
-      <div key={request.id} className="request bg-white mb3">
-        <div className="fl w-25">
-          <WorkFixedPanel work={request.work}/>
+      <div key={this.state.request.id} className="bg-white mb3">
+        <div className="flex justify-between w-100 items-center bb b--light-gray bt-0 bl-0 br-0">
+          { this.props.artist ? 
+            (<BuyerSnapshot buyer={this.state.request.buyer} />) : 
+            (<ArtistSnapshot artist={this.state.request.artist} />)
+          }
+          <div className="pa3 flex-grow-1">
+            <h5>{isRental ? "Rental Start" : ""}</h5>
+            <p>{startDate}</p>
+          </div>
+          <div className="pa3 flex-grow-1">
+            <h5>{isRental ? "Rental End" : ""}</h5>
+            <p>{endDate}</p>
+          </div>
+          <div className="pa3">
+            <h5>Price</h5>
+            <p>{"$" + convertToCurrency(this.state.request.receipt.price)}</p>
```

Comment:
For some reason it's only showing 1 digit after the decimal, e.g. `$100.0` 🤔

Reply:
Fixed in `currency.js`!